packages/components/addon/components/hds/app-footer/copyright.hbs

```diff
@@ -0,0 +1,9 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::AppFooter::Item class="hds-app-footer__item--copyright" ...attributes>
```

Comment:
not needed anymore now
```suggestion
<Hds::AppFooter::Item ...attributes>
```

Reply:
I moved it so I had to keep a class name. I can change it again if needed though.